src/init.ts

```diff
@@ -155,8 +155,7 @@ async function generateTsConfig(options: Options): Promise<void> {
   const tsconfig = formatJson({
     extends: './node_modules/gts/tsconfig-google.json',
     compilerOptions: {rootDir: '.', outDir: 'build'},
-    include: ['src/*.ts', 'src/**/*.ts', 'test/*.ts', 'test/**/*.ts'],
-    exclude: ['node_modules']
+    include: ['src/*.ts', 'src/**/*.ts', 'test/*.ts', 'test/**/*.ts']
```

Comment:
Side note. We say `main: index.js` when we say our source files are in these directories... well...

Reply:
That's because we generate the `package.json` using `npm` defaults hence:

https://github.com/ofrobots/ts-style/blob/4290d772df259e886728770f3cb77fba8eb8a2db/src/init.ts#L199

We should refine this to better match the expected project layout.